src/app/common/default.controller.js

```diff
@@ -0,0 +1,52 @@
+
+(function() {
+  'use strict';
+  angular
+    .module('healthGuide')
+    .controller('defaultController', defaultController);
+
+  function defaultController(
+                      localForageService,
+                      sectionsService,
+                      sidenavService,
+                      userService
+                      ) {
+      "ngInject";
+      /*jshint validthis: true*/
+      var self = this;
+      self.state = 'default';
+
+      self.toggleSideNav = sidenavService.toggleList;
+      self.openLeftMenu = sidenavService.openLeftMenu;
+      self.closeSidenav = sidenavService.closeSidenav;
+
+      self.user = userService.user;
+      self.data = sectionsService.data[self.state];
+      self.calculate = calculate;
+
+      activate();
+
+      function activate() {
+        self.closeSidenav();
+        if (_.isEqual(userService.user, userService.mockUser)) {
```

Comment:
this is a hack, dude
if something should be done after,
it should be done after,
not after some period.

run does not work for your case and actually it is for app settings only
you can get user in controller;)